src/Jackett.Common/Indexers/BroadcasTheNet.cs

```diff
@@ -95,21 +95,33 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
                 btnResults = (int)TorznabCaps.LimitsDefault;
             var btnOffset = query.Offset;
             var releases = new List<ReleaseInfo>();
+            var searchParam = new Dictionary<string, string>();
 
-            // If only the season is searched for then change format to match expected format
-            var seasonOnlyMatch = new Regex(@".*\s[Ss]{1}\d{2}(?<![Ee]{1}\d{2,3})?$").Match(searchString);
+            // If only the season/episode is searched for then change format to match expected format
+            var seasonOnlyMatch = new Regex(@"(.*)\s[Ss]{1}\d{2}(?<![Ee]{1}\d{2,3})$").Match(searchString);
+            var seasonEpisodeMatch = new Regex(@"(.*)\s[Ss]{1}\d{2}[Ee]{1}(\d{2,3})$").Match(searchString);
```

Comment:
Do you need the regular ex? I think Jackett does this job for you. Check the values of query.episode and query.season. You can do something like:
```
if (query.season > 0 && query.episode == 0) // season
else if (query.season > 0 && query.episode > 0) // season + episode
else // none
```

Reply:
Thanks, I've made the necessary modification and tested it.